python/dgl/distributed/graph_partition_book.py

```diff
@@ -923,6 +982,34 @@ def etypes(self):
         """
         return self._etypes
 
+    @property
+    def canonical_etypes(self):
```

Comment:
Expose `canonical_etypes` and `to_canonical_etype` to the class docstring https://docs.dgl.ai/api/python/dgl.distributed.html#graph-partition-book .

Reply:
`canonical_etypes` exposed. `to_canonical_etype` is restricted to be used internally within `RangePartitionBook`.